Sources/Compiler/AST/Expr/NameExpr.swift

```diff
@@ -1,17 +1,25 @@
 /// A name denoting an object.
 public struct NameExpr: Expr {
 
-  public enum Domain: Equatable, Codable {
+  /// The expression of a name's qualification, or absence thereof.
```

Comment:
```suggestion
  /// A name's qualification
```

“none” is a fine qualification.